crates/turborepo-lib/src/run/task_id.rs

```diff
@@ -99,6 +99,10 @@ impl<'a> TaskId<'a> {
         }
     }
 
+    pub fn as_github_task_id(&self) -> String {
+        format!("{}:{}", self.package, self.task)
+    }
```

Comment:
Won't die on this hill, but this doesn't feel like it needs to be defined on `TaskId`, I think it would better be defined as a function in `turborepo-ci` maybe on the `Vendor` type.